proposals/static-delegates.md

```diff
@@ -0,0 +1,74 @@
+# Static Delegates
+
+* [x] Proposed
+* [ ] Prototype: Not Started
+* [ ] Implementation: Not Started
+* [ ] Specification: Not Started
+
+## Summary
+[summary]: #summary
```

Comment:
It allows you to link to the sub-section of the document (this is based on the [proposal-template.md](https://github.com/dotnet/csharplang/blob/master/proposals/proposal-template.md)).